docs/administration/back_office/search_sorting.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Change file name to "customize_search_sorting.md" to be consistent with what is visible in TOC.

Reply:
Done in 5527c36956be0e2296ad36fbdfb8b6386afec4dc